source/includes/_tipser-react-elements.md

```diff
@@ -184,65 +202,257 @@ Components are the building blocks of Tipser Elements. Any components need to be
+  carousel
+  imgSize={"small"}
+/>
 ```
 
-prop name  | description | type  | required | default value 
------------|-------------|-------|----------|--------------
-collectionId | [where to find](#getting-tipser-product-id-and-collection-id) | string | true | none 
-carousel | enables carousel display | boolean | false | false
-imgSize | changes the size of single product tile | string ('small', 'medium' or 'large') | false | none
+| prop name    | description                                                   | type                                  | required | default value |
+| ------------ | ------------------------------------------------------------- | ------------------------------------- | -------- | ------------- |
+| collectionId | [where to find](#getting-tipser-product-id-and-collection-id) | string                                | true     | none          |
+| carousel     | enables carousel display                                      | boolean                               | false    | false         |
+| imgSize      | changes the size of single product tile                       | string ('small', 'medium' or 'large') | false    | none          |
+
+## `Product`
 
-## `Product` 
-Element that renders the product title based on Tipser [productId](#getting-tipser-product-id-and-collection-id) passed to the `productId` prop. 
+Element that renders the product title based on Tipser [productId](#getting-tipser-product-id-and-collection-id) passed to the `productId` prop.
 
 The `Product` element supports two display modes (controlled by `viewMode` prop):
 
 - Large product title with variant selector and "add to cart" button, if `viewMode="full"` (default)
 - Small product title which when clicked opens a product dialog, if `viewMode="compact"`
 
-prop name  | description | type  | required | default value 
------------|-------------|-------|----------|--------------
-productId | [where to find](#getting-tipser-product-id-and-collection-id) | string | true | none 
-viewMode | enables full or compact product display | string ('full', 'compact') | false | 'full'
+| prop name | description                                                   | type                       | required | default value |
+| --------- | ------------------------------------------------------------- | -------------------------- | -------- | ------------- |
+| productId | [where to find](#getting-tipser-product-id-and-collection-id) | string                     | true     | none          |
+| viewMode  | enables full or compact product display                       | string ('full', 'compact') | false    | 'full'        |
 
 _example:_
 
 ```jsx
-<Product productId="5c751cf82d3f3b0001bcec8c" viewMode={'compact'} />
+<Product productId="5c751cf82d3f3b0001bcec8c" viewMode={"compact"} />
 ```
 
 [Learn more](#product-element).
 
+## `Modular Product`
+
+For a more customazible and featured <code>Product</code> component you can use
```

Comment:
"and featured <code>Product</code> component" -> "and **feature-rich version of the** <code>Product</code> component"